README.md

```diff
@@ -27,14 +27,14 @@ To get a local copy up and running follow these steps.
 
    ```shell
    git clone git@github.com:Thomann1992/devops_project.git devops_projekt
-   ```
-   
+   ````
```

Comment:
Use only three `` ` ``s to end a code block. Doesn't the Markdown linter complain about this?